blockchain/chainio.go

```diff
@@ -38,6 +39,27 @@ var (
 	// unspent transaction output set.
 	utxoSetBucketName = []byte("utxoset")
 
+	// thresholdBucketName is the name of the db bucked used to house cached
```

Comment:
bucked -> bucket

The same typo is repeated in several of the variable comments below.